src/docs/user/examples/CriticalSection.xml

```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
should these workflows also be in the sample bucket in the proactive-examples project?

Reply:
Maybe a dedicated bucket ?